### DISCUSSION TITLE
Update README to add a section describing what your lib is doing

### DISCUSSION THREAD

**Discussion by bennekrouf** [glowroot/glowroot#1231] [Ideas]
I am really wondering what is this project. A short introduction sentences would help.

Thanks

**Comment by nowheresly** [comment]
Hi
you should consider having a look at the website
https://glowroot.org/

**Comment by Never-lab** [comment]
## Triage

**Ideas / documentation** — not a code bug.

Fair request. `main` branch README now opens with a short product description, deployment pointers, and community channels (post-revival work). If still insufficient, a "What is Glowroot?" paragraph could be expanded.

**Verdict:** convert to **Discussions → Ideas** (or close if README on `main` is enough once released).

---
*Community triage*
